src/components/ButtonsTable.js

```diff
@@ -1,35 +1,58 @@
 import React from 'react';
-import Button from './Button';
+import PropTypes from 'prop-types';
+import { Button } from './Button';
 
-class ButtonsTable extends React.Component {
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
- [ ] Kindly remove the disable linters, as linter rules help guide us to write industry standard code.